galasa-ui/messages/en.json

```diff
@@ -194,7 +193,7 @@
     "noTags": "No tags were associated with this test run."
   },
   "TestRunDetails": {
-    "title": "Test Run:",
+    "title": "Test Run:({runName})",
```

Comment:
Can we remove the surrounding `()` from the test run name and add a space between "Test Run:" and the name?